src/app/pages/question/question.component.html

```diff
@@ -27,4 +27,5 @@
   *ngIf="(modalWindowState$ | async)" 
   (confirm)="handleModalResponse($event)" 
   [data]="(modalWindowData$ | async)!"
-/>
\ No newline at end of file
+/>
+<app-ui-notification [show]="(isNotification$ | async)!" />
```

Comment:
if you need to show the component, it doesn't need to be a input to this component, because your notification component only show/hide itself and has no logic related to 'show'. In such case you can remove 'show' property from it and just apply *ngIf to app-ui-notification itself right here.